volatility3/framework/interfaces/symbols.py

```diff
@@ -167,6 +167,20 @@ def set_type_class(self, name: str, clazz: Type[objects.ObjectInterface]) -> Non
         """
         raise NotImplementedError("Abstract method set_type_class not implemented yet.")
 
+    def try_set_type_class(self, name: str, clazz: Type[objects.ObjectInterface]) -> bool:
```

Comment:
I'm kind of ok with this idea, although I'm not keen on the `try_` name but I'm not sure what would be better, maybe something about `optional`?  It just seems as though the method name doesn't explain what happens in the case of failure.

Reply:
renamed it